cloudos_cli/jobs/job.py

```diff
@@ -1133,16 +1137,41 @@ def clone_or_resume_job(self,
             nextflow_version not in ['22.11.1-edge', 'latest']:
             print("Azure workspace only uses Nextflow version 22.11.1-edge, option '--nextflow-version' is ignored.\n")
 
+
         # Override branch if provided
         # sometimes revision is missing from the 'request-payload' API, make sure is present
         if 'revision' not in cloned_payload or not cloned_payload.get('revision'):
```

Comment:
```suggestion
        if 'revision' not in cloned_payload:
```
Why to test here this two times?

Reply:
The comment you mention is for a different implementation I see. In this particular case, `'revision' not in cloned_payload` will be `True` if the dict does not contain that key (or fail with a TypeError if it is not an iterable). The second expression after the or is not necessary, because it will test exactly the same